pyproject.toml

```diff
@@ -19,7 +19,7 @@ dynamic = [
 readme = "README.rst"
 requires-python = ">=3.10"
 dependencies = [
-    "numpy>=1.21.6",
+    "numpy>=1.21.6,<2.4.0",
```

Comment:
Can you make a ticket after this release to put upper bounds on dependencies for the first 2 version numbers?  And another ticket to speak with @marcio-melendez about removing the dependencies causing the numpy 2.4.0 error.